deploy/install_common_libraries.sh

```diff
@@ -23,7 +23,7 @@ source ~/.profile
 # Debian Bookdown does not like pip install wthout a virtualenv, so do apt installs instead
 # Also need gstreamer1.0-libcamera, as the libcamerasrc gst element has moved
 source /etc/os-release
-if [ "$ID" == "debian" ] && [ "$VERSION_CODENAME" == "bookworm" ]; then
+if [ "$ID" == "debian" ] || [ "$VERSION_CODENAME" == "bookworm" ]; then
```

Comment:
Use ``if [ "$ID" == "debian" || "$ID" == "raspian" ] && [ "$VERSION_CODENAME" == "bookworm" ]; then``

That way it'll detect both variants

Reply:
I have update the PR, and it works on my pi 3B+